src/scripts/apiScripts.js

```diff
@@ -0,0 +1,50 @@
+import cookie from 'react-cookies'
+
+export const BACK_AVAILABLE = ( process.env.REACT_APP_BACK_API_AVAILABLE == 'true' ? true : false ); 
+export const API_KEY = process.env.REACT_APP_BACK_API_KEY;
+export const BACK_HOST_NAME = process.env.REACT_APP_BACK_API_HOST_NAME;
+                             
+export const loginAPI = async (username,password) => {
+  const query_string='?username='+username+'&password='+password;
+  
+  if(!BACK_AVAILABLE) {
+    return {username:'User',token:''};
+  }
+
+  return fetch(BACK_HOST_NAME+'sign_in'+query_string, {
```

Comment:
Can this be declared as a separate function? 
`return userSignIn(query_string){...}`